nf_core/modules/lint/main_nf.py

```diff
@@ -212,22 +212,18 @@ def check_process_section(self, lines):
 
     for l in lines:
         l = l.strip()
-        l = l.replace('"', "")
-        l = l.replace("'", "")
+        l = l.lstrip('"')
+        l = l.lstrip("'")
         if re.search("bioconda::", l):
```

Comment:
It would also be nice to move this down to sit alongside the loop that uses this variable, whilst we're here..